crates/api/src/builder.rs

```diff
@@ -15,6 +15,10 @@ pub struct Builder {
     /// The [agent::Verifier] to use for this Kitsune2 instance.
     pub verifier: agent::DynVerifier,
 
+    /// The [kitsune::KitsuneFactory] to be used for creating
+    /// [kitsune::Kitsune] kitsune-level kitsune2 module instances.
```

Comment:
kitsune kitsune kitsune
```suggestion
    /// [kitsune::Kitsune] module instances.
```